libzmq/src/auth/server.rs

```diff
@@ -46,10 +46,10 @@ impl fmt::Display for StatusCode {
     }
 }
 
-#[derive(Debug, Fail)]
-#[fail(display = "unable to parse status code")]
+#[derive(Debug, Error)]
+#[error("unable to parse status code")]
 #[doc(hidden)]
-pub struct StatusCodeParseError(());
```

Comment:
Whats the reasoning behind making this public?

Reply:
Assumed it wasn't part of public API, will fix